book/src/ui-usage.md

```diff
@@ -1,73 +1,105 @@
 
-<img src="imgs/ui-validator-balance2.png" width="48%" alt="validator-balance2" />
+<img src="imgs/ui-validator-balance2.png" width="48%" style="margin-bottom: 25px" alt="validator-balance2" />
 
-## Hardware Usage and Device Diagnostics
+### Hardware Usage and Device Diagnostics
 
 The hardware usage component gathers information about the device the Beacon Node is currently running. It displays the Disk usage, CPU metrics and memory usage of the Beacon Node device. The device diagnostics component provides the sync status of the execution client and beacon node.
 
 <img height="350" src="imgs/ui-hardware.png" style="display: inline; float: left; margin-right: 25px" alt="hardware" />
 
 <img height="350" src="imgs/ui-device.png" alt="device" />
 
-## Log Statistics
+### Log Statistics
+
+The log statistics present an hourly combined rate of critical, warning, and error logs from the validator client and beacon node. This analysis enables informed decision-making, troubleshooting, and proactive maintenance for optimal system performance. You can view the full log outputs in the logs page by clicking `view all` at the top of the component.
 
-The log statistics present an hourly combined rate of critical, warning, and error logs from the validator client and beacon node. This analysis enables informed decision-making, troubleshooting, and proactive maintenance for optimal system performance.
+<img height="350" src="imgs/ui-dash-logs.png" style="margin-bottom: 50px" alt="log" />
 
-<img height="350" src="imgs/ui-dash-logs.png" alt="log" />
+________________________________________________________________________________________________________________________________
 
-## Validator Management
+# Validator Management
 
 Siren's validator management view provides a detailed overview of all validators with options to deposit to and/or add new validators. Each validator table row displays the validator name, index, balance, rewards, status and all available actions per validator.
 
 ![validator-management](imgs/ui-validator-management.png)
 
-## Validator Modal
+### Validator Modal
 
 Clicking the validator icon activates a detailed validator modal component. This component also allows users to trigger validator actions and as well to view and update validator graffiti. Each modal contains the validator total income with hourly, daily and weekly earnings estimates.
 
-<img height="450" src="imgs/ui-validator-modal.png" alt="ui-validator-modal" />
+<img height="450" src="imgs/ui-val-modal.png" alt="ui-validator-modal" />
 
-## Settings
+### Validator BLS Execution
+When Siren detects that your validator is using outdated BLS credentials, it will temporarily block any further actions by the validator. You can identify if your validator does not meet this requirement by an `exclamation mark` on the validator icon or a message in the validator modal that provides instructions for updating the credentials.
```

Comment:
The exclamation mark mentions that validators cannot be exited without updating the withdrawal credentials. But it is possible to exit without updating the withdrawal credentials, it is just that the validator will not receive the staked funds upon exited. Will Siren allow exiting without updating the withdrawal credentials?

```suggestion
When Siren detects that your validator is using outdated BLS withdrawal credentials, it will temporarily block any further actions by the validator. You can identify if your validator does not meet this requirement by an `exclamation mark` on the validator icon or a message in the validator modal that provides instructions for updating the credentials.
```

Reply:
Siren will not allow access to exit validator action without updating withdrawal credentials. I mean validators cannot exit through siren without them.

Reply:
Just thought it would be nice to be able to exit without updating withdrawal credentials, since the only requirement is having the keystore file and calling the exit API